pandas/tests/plotting/test_frame.py

```diff
@@ -144,8 +145,26 @@ def test_plot(self):
         result = ax.axes
         assert result is axes[0]
 
-    # GH 15516
+    def test_integer_array_plot(self):
+        # GH 25587
+        arr = integer_array([1, 2, 3, 4], dtype="UInt32")
```

Comment:
will a plot actually work if there are nulls?

Reply:
@jreback  do you mean there are `nan`s or `None`s in the integer_array? It will also can be plotted.

In fact, `np.asarray` will transform `None` as `np.nan`, and the result dataframe still can be plotted.

```
>>> a = integer_array([1,2,3,None], dtype="Int64")
>>> a
<IntegerArray>
[1, 2, 3, NaN]
Length: 4, dtype: Int64
>>> np.asarray(a)
array([1, 2, 3, nan], dtype=object)
```